tapir/shifts/management/commands/apply_shift_cycle_start.py

```diff
@@ -0,0 +1,45 @@
+import datetime
+
+from django.core.management.base import BaseCommand
+from django.db.models import Max
+
+from tapir.shifts.models import ShiftCycleEntry, ShiftTemplateGroup, Shift
+from tapir.utils.shortcuts import get_monday
+
+
+class Command(BaseCommand):
+    help = "If a new cycle has started, remove one shift point from all active members."
+
+    def handle(self, *args, **options):
```

Comment:
Maybe a quick comment to explain what is happening here? It's pretty specific to how SuperCoop works.